src/v/cluster/topics_frontend.cc

```diff
@@ -317,10 +317,11 @@ topics_frontend::dispatch_create_non_replicable_to_leader(
         [topics, timeout](controller_client_protocol cp) mutable {
             return cp.create_non_replicable_topics(
               create_non_replicable_topics_request{std::move(topics), timeout},
-              rpc::client_opts(model::timeout_clock::now() + timeout));
+              rpc::client_opts(rpc::no_timeout));
```

Comment:
This ignores the timeout the user passed in and always uses `no_timeout` instead

Reply:
Vadim says I need to use no_timeout here to pass tests

Reply:
I wrote about `no_timeout` only in [link](https://github.com/vectorizedio/redpanda/pull/3119#discussion_r764357128) I used it only to pass test in CI. It is not production solution.

```
static constexpr timeout_clock::time_point no_timeout  = timeout_clock::time_point::max();
```

As you can see `no_timeout` - set time_point::max(). But user can want to set normal timeout (several seconds, for example). And if you do not use it user will wait more than they want